docs/arcaflow/workflows/schemas.md

```diff
@@ -19,7 +19,9 @@ Schemas that are not composed within an ObjectSchema do not have an `Object ID`.
 | OneOfString   | one_of_string   |
 | OneOfInt      | one_of_int      |
 
-`ListSchema`s use the `ListSchema` `TypeID` with the schema name of their constituent element, or item value. `ListSchema` names are prepended with `list` until they reach a non-`ListSchema` item type. These names are joined with a single underscore, `_`.
+`ListSchema`s use the `ListSchema` `TypeID` with the schema name of their constituent element, or item value. `ListSchema` names are prefixed with `list` until they reach a non-`ListSchema` item type. These names are joined with a single underscore, `_`.
+

```

Comment:
I would avoid pluralizing monospace items (and, if it's unavoidable (as perhaps it is in lines 49 and 50), I would use `'s` to pluralize them, ).  We could rephrase the first sentence as,
> The name of a `ListSchema` is the name of the schema of its element type prefixed with `list_`.

And, I would rephrase the second sentence like,
> For lists of lists, the schema name is the name of the inner list schema prefixed with an additional `list_`.

I don't think we need the phrase starting with "until they reach".  (I think the table makes clear what happens when there is a deeply-nested list.)  And, including the underscore in the stated prefix allows us to omit the last sentence.

Reply:
changed